src/main/java/gomedic/logic/commands/CommandResult.java

```diff
@@ -58,6 +58,8 @@ public boolean equals(Object other) {
         }
 
         CommandResult otherCommandResult = (CommandResult) other;
+        System.out.println(feedbackToUser);
+        System.out.println(((CommandResult) other).feedbackToUser);
```

Comment:
Is this System.out.println required or you use it for your own debugging purpose?

Reply:
oh yeah gg